web/src/app/converters/proto-model-converter.ts

```diff
@@ -99,17 +103,22 @@ export function jobToDocument(job: Job): DocumentData {
       index: job.index,
       name: job.name,
       style: new Pb.Style({color: job.color}),
-      tasks: tasksToDocument(job.tasks?.toList() || List([])),
+      tasks: toTasksMessage(job.tasks?.toList() || List([])),
```

Comment:
Why not just handle the list conversion here and have the method just be `toTaskMessage()`?